### ISSUE TITLE
`crypto_stream_chacha20_ietf_xor_ic()` always uses the zero-filled array key.

### ISSUE THREAD

**Issue by simnalamburt** [jedisct1/libsodium.js#248]
The function `crypto_stream_chacha20_ietf_xor_ic()` ignores the key parameter, always treat the key parameter as zero-filled array. This bug could lead to serious security vulnerabilities, so quick action is necessary.

I tried to fix it by myself but I have no idea what the cause is. [`wrapper/symbols/crypto_stream_chacha20_ietf_xor_ic.json`](https://github.com/jedisct1/libsodium.js/blob/master/wrapper/symbols/crypto_stream_chacha20_ietf_xor_ic.json) looked fine for me.

### Minimal bug reproduction using Node.js
```js
// libsodium-wrappers-sumo v0.7.6
const sodium = require('libsodium-wrappers-sumo')

main()

async function main() {
  await sodium.ready

  // NOTE: Whatever key you choose, result will be the same as using the zero key
  const key = sodium.crypto_stream_chacha20_keygen()
  // const key = new Uint8Array(32)
  // const key = Uint8Array.from('01234567890123456789012345678901', c => c.charCodeAt())
  // const key = Uint8Array.from('yoloswagyoloswagyoloswagyoloswag', c => c.charCodeAt())

  const result = sodium.crypto_stream_chacha20_ietf_xor_ic(
    new Uint8Array(64),
    Uint8Array.from('Hello,world!', c => c.charCodeAt()),
    0,
    key)

  console.log(result)
}
```

### Minimal bug reproduction in browser
```html
<!doctype html>
<html>
<head><meta charset="UTF-8"></head>
<body>

<script src="https://cdn.jsdelivr.net/gh/jedisct1/libsodium.js@0.7.6/dist/browsers-sumo/sodium.js"></script>
<script>
main()

async function main() {
  await sodium.ready

  // NOTE: Whatever key you choose, result will be the same as using the zero key
  const key = sodium.crypto_stream_chacha20_keygen()
  // const key = new Uint8Array(32)
  // const key = Uint8Array.from('01234567890123456789012345678901', c => c.charCodeAt())
  // const key = Uint8Array.from('yoloswagyoloswagyoloswagyoloswag', c => c.charCodeAt())

  const result = sodium.crypto_stream_chacha20_ietf_xor_ic(
    new Uint8Array(64),
    Uint8Array.from('Hello,world!', c => c.charCodeAt()),
    0,
    key)

  console.log(result)
}
</script>

</body>
</html>
```

**Comment by simnalamburt** [comment]
@jedisct1 This issue is still reproduced in version 0.7.7. Can you reopen this issue?

![image](https://user-images.githubusercontent.com/4435445/89322483-3ab10a00-d6bf-11ea-9d3f-bb32dfcc4123.png)

**Comment by jedisct1** [comment]
Sorry, I can't see what's on that picture.

The fix is correct, but the packages published to npm may be identical to the previous version. New packages have been published.

Creating these npm packages is currently very tedious, due to the multiple packages and their dependencies. You know JavaScript way better than I do, so your help at that purpose would be welcome!

**Comment by simnalamburt** [comment]
I've just confirmed that this issue has been fixed in version 0.7.8! Thanks for your work.

"Yarn Workspace" or "Lerna" seems to help in such situations. I'll try to contribute if I have time to spare.
